src/components/ProfileDropdown.jsx

```diff
@@ -84,7 +81,7 @@ export default function ProfileDropdown(props) {
 
   return (
     <>
-      <Avatar src={photo} onClick={handleClick} />
+      <Avatar src={photo} onClick={handleClick} className={classes.medium} />
```

Comment:
The `className={classes.medium}` might not be needed if a common width is set in the `LoginButton`.

Reply:
+1. If `minWidth` is set on the element that contains both the login button and the profile dropdown then the size of the Avatar does not need to change.

Also, this new size is making the header bar slightly thicker when loaded so it introduces a sizing glitch when it goes from loading -> medium avatar shown.

Reply:
Ok yea that did work

Reply:
Its not making a difference since now the Login button and the avatar are always below the links

Reply:
That was what the changes that messed up the iPad view was suppose to fix but since you wanted them undone it no longer fixes that sizing between the links and button/avatar